src/traces/bar/cross_trace_calc.js

```diff
@@ -770,8 +790,15 @@ function collectExtents(calcTraces, pa) {
             di.p1 = di.p0 + di.w;
             di.s0 = di.b;
             di.s1 = di.s0 + di.s;
+
+            var sMin = Math.min(di.s0, di.s1);
+            var sMax = Math.max(di.s0, di.s1);
+
+            sMinByPos[di.p] = (di.p in sMinByPos) ? Math.min(sMinByPos[di.p], sMin) : sMin;
```

Comment:
Accessing `p` property using dot operator multiple times may be a bit slow.
Let's store the value in a variable once and use that.
```js
var p = di.p
...
```